include/dra.h

```diff
@@ -621,6 +621,10 @@ typedef struct {
     /* 8003C8B4 END*/
 } GameEngine; /* size=0x140 */
 
+typedef struct {
+    u8 data[0x278];
+} UnkStruct11;
+
```

Comment:
Hello and welcome to the project!
Just a small nit to keep our project structure consistent.

This struct should go inside unkstruct.h as we are in the process of identifying and merging them all, work it's being done while we import real data to figure out the rest of the project structure so we are putting them in that file in the meantime. It's name should be Unkstruct_8013B160 to state which symbol it corresponds to. We also state the name in both ends in case there's a pointer to itself which we don't know.
```suggestion
typedef struct Unkstruct_8013B160  {
    u8 data[0x278];
} Unkstruct_8013B160;
```
I would also recommend to read docs/style.md to familiarize yourself to our style :)

Thank you for contributing to this Dream project, we are happy to see you working among us!

Reply:
Thanks, I believe the comments are addressed now. If it looks good I can rebase.

Reply:
Looks good to me!